### DISCUSSION TITLE
Can you modify the uniqueness of the client after opening

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#2864] [General]
Every time double-click the icon to open a program, it is troublesome in the remote process, you need to switch constantly, the remote only keeps a remote dialog box is the most convenient, before 1.1.9 remote dialog box did not merge is fine

**Comment by youhers** [comment]
Open the client repeatedly Multiple dialogs pop up This is really troublesome I hope it can be modified in subsequent versions Thank you

**Comment by youhers** [comment]
When running, can you set to check whether the exe is open, such as already open, call the program that has been opened

**Comment by rustdesk** [comment]
@Kingtous it is time to consider this indeed for Windows, Mac has no this problem.

**Reply by youhers** [reply]
Thank you

**Reply by rustdesk** [reply]
When main window is closed, but remote window is open, it should create new main window. When main window is open or hidden or minized, it should bring it top. Make the behavior friendly.

**Reply by youhers** [reply]
Yes, currently repeated opening will produce multiple client programs in practical applications is really troublesome

**Reply by Kingtous** [reply]
> Yes, currently repeated opening will produce multiple client programs in practical applications is really troublesome

I agree, lets get the job done.
